scripts/stage

```diff
@@ -23,6 +23,7 @@ IMAGE_NAME=""
 AWS_PROFILE=""
 S3_BUCKET=""
 S3_ACL_OVERRIDE=""
+DEVCON_ZIP_URI="s3://amazon-ecs-devcon/devcon-latest.zip"
```

Comment:
Can you use `devcon-3a3f98688359aa1b905147831f910218.zip` instead?  I'd like a strongly-versioned artifact here.